### ISSUE TITLE
Could not load file or assembly Newtonsoft.Json version 7.0.0.0

### ISSUE THREAD

**Issue by DavidRooney** [DotNetAnalyzers/StyleCopAnalyzers#2255]
Just added the stylecop analyzer nuget package to my VS 2015 and getting lots of the same error below:

> Warning	AD0001	Analyzer 'StyleCop.Analyzers.OrderingRules.SA1208SystemUsingDirectivesMustBePlacedBeforeOtherUsingDirectives' threw an exception of type 'System.IO.FileNotFoundException' with message 'Could not load file or assembly 'Newtonsoft.Json, Version=7.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The system cannot find the file specified.'.

Cannot find any way to fix the issue. I have tried replacing the newtonsoft.json.dll in the package directory with the net45 version and this did not make the warning go away.

Any help?

Thanks

**Comment by sharwell** [comment]
1. What version of StyleCop Analyzers have you installed?
1. Are you using any other analyzers? If so, which one(s)?

**Comment by RodionovDmitry** [comment]
Just had the same issue. Looks like Newtonsoft.Json package installs with errors because of the script execution policy.
So I run windows PowerShell: set-ExecutionPolicy Unrestricted -scope CurrentUser
Uninstalled both StyleCop.Analyser and Newtonsoft.Json packages
Installed StyleCop.Analyser (latest stable) and now it is working.

**Comment by sharwell** [comment]
> Looks like Newtonsoft.Json package installs with errors because of the script execution policy.

This is confusing to be, because you shouldn't need to "install" Newtonsoft.Json to use StyleCop Analyzers.

**Comment by vweijsters** [comment]
💭  I think he means that the install.ps1 file executed during NuGet package install is causing the problem. A quick search on Google revealed that more people have this issue and setting the execution policy solves it.

**Comment by sharwell** [comment]
Our install scripts are signed, but I just noticed the certificate has since expired. All packages I could find, including those from Microsoft, no longer contain signatures in the scripts. It appears the problem was fixed in a more recent release of the NuGet extension for Visual Studio.

@DavidRooney @RodionovDmitry If you upgrade the NuGet Package Manager extension in Visual Studio, it  should install the package without problems.
